proposals/0004-FaultTolerance.md

```diff
@@ -89,15 +89,19 @@ Connection connect = execution.with(cb).run(this::connect);
 
 When 3 of 10 execution failures occurs on a circuit breaker, the circuit is opened and further execution requests fail with `CircuitBreakerOpenException`. After a delay of 1 min, the circuit is half-opened and trail executions are attempted to determine whether the circuit should be closed or opened again. If the trial executions exceed a success threshold 5, the breaker is closed again and executions will proceed as normal.
 
-### Bulkhead
+### BulkHead
```

Comment:
The `bulkhead` spelling here comes from the original pattern in *Release It!* as well as [Fowler's writing](https://martinfowler.com/articles/microservices.html) on the subject. Similar with `timeout`.